x-pack/test_serverless/api_integration/config.base.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrConfigProviderContext } from '@kbn/test';
+import { services } from './services';
+import type { CreateTestConfigOptions } from '../shared_types';
+
+export function createTestConfig(options: CreateTestConfigOptions) {
+  return async ({ readConfigFile }: FtrConfigProviderContext) => {
+    const xpackApiIntegrationTestsConfig = await readConfigFile(
+      // eslint-disable-next-line @kbn/imports/no_boundary_crossing
+      require.resolve('../../test/api_integration/config.ts')
```

Comment:
+1 on this, I've had to debug configuration changes a few times and the long inheritance chain adds a lot of complexity.

Reply:
That's a fair point and also gives us the opportunity to start with a minimal set of ES and Kibana server arguments (and only add to them where needed instead of blindly taking over everything that we have in stateful). Updated in f012285.